removed N_nash_subsruface bmi parameter (#131)
NOAA-OWP/cfe: configs/README.md

## Configuration File

> Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. The asterisk (*) denotes calibratable parameters. The has (#) shows the parameters only used with the Nash Cascade-based runoff.

Reviewer: ```suggestion
Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. 

**Key**:

<sup>*</sup>: denotes parameter is calibratable

<sup>1</sup>: denotes the parameters only used when `surface_runoff_scheme` is `NASH_CASCADE`.
```

This renders as:

Example configuration files are provided in this directory. To build and run the given examples see the instructions [here](https://github.com/NOAA-OWP/cfe/blob/master/INSTALL.md). A detailed description of the parameters for model configuration (i.e., initialize/setup) is provided below. 

**Key**:

<sup>*</sup>: denotes parameter is calibratable

<sup>1</sup>: denotes the parameters only used when `surface_runoff_scheme` is `NASH_CASCADE`.

----------

It might be easiest to digest if the Key section is moved after the parameter table.

Reviewer 2: @ajkhattak, if you don't implement the change here from @aaraney, please change "has" to "hash".

| verbosity | *int* | `0`-`3`  |   | optional |   |  prints various debug and bmi info (defaults to 0) |
| surface_water_partitioning_scheme | *char* | `Xinanjiang` or `Schaake`  |  | parameter_adjustable | infiltraton excess |    |
| surface_runoff_scheme | *char* | GIUH or NASH_CASCADE | | parameter_adjustable | surface runoff | also supports 1 for GIUH and 2 for NASH_CASCADE; default is GIUH |

> | #N_nash_surface | *int* |   |   | parameter_adjustable | surface runoff | number of Nash reservoirs for surface runoff   |

Reviewer: ```suggestion
| N_nash_surface<sup>1</sup> | *int* |   |   | parameter_adjustable | surface runoff | number of Nash reservoirs for surface runoff   |
```

| N_nash_surface<sup>1</sup> | *int* |   |   | parameter_adjustable | surface runoff | number of Nash reservoirs for surface runoff   |

| #K_nash_surface | *double* |   | 1/hour [h^-1]  | parameter_adjustable | surface runoff | Nash Config param for surface runoff   |
| #nash_storage_surface | 1D array (*double*) |   | meters [m]  | parameter_adjustable | surface runoff | Nash Config param; reservoir surface storage; default is zero storage |
| #nsubsteps_nash_surface | *int* |   |   | parameter_adjustable | surface runoff | optional (default = 10); number of subtimstep for Nash runoff |

> | #*Kinf_nash_surface | *double* |   | 1/hour [h^-1] | parameter_adjustable | surface runoff | optional (default = 0.05); storage fraction per hour that moves from reservoirs to soil |

Reviewer: ```suggestion
| Kinf_nash_surface<sup>*,1</sup> | *double* |   | 1/hour [h^-1] | parameter_adjustable | surface runoff | optional (default = 0.05); storage fraction per hour that moves from reservoirs to soil |
```

| Kinf_nash_surface<sup>*,1</sup> | *double* |   | 1/hour [h^-1] | parameter_adjustable | surface runoff | optional (default = 0.05); storage fraction per hour that moves from reservoirs to soil |
